packages/tinacms/src/toolkit/fields/components/text-field.tsx

```diff
@@ -23,6 +24,7 @@ export const BaseTextField = React.forwardRef<
       className={`${textFieldClasses} ${
         disabled ? disabledClasses : ''
       } ${className}`}
+      dir={getDirection(rest.value)}
```

Comment:
I don't think we need this since we are setting the direction in the `wrap-field-with-meta.tsx` file.